build.gradle

```diff
@@ -18,13 +18,11 @@ buildscript {
     repositories {
         maven { url "https://plugins.gradle.org/m2/" }
     }
-    dependencies {
-        classpath 'gradle.plugin.org.jetbrains:gradle-intellij-plugin:0.0.43'
-    }
 }
 
 plugins {
     id 'net.researchgate.release' version '2.3.4'
+    id 'org.jetbrains.intellij' version '0.1.5'
```

Comment:
Could be. The new "plugins" syntax is new to gradle > 2.1: https://docs.gradle.org/2.1/release-notes#new-and-noteworthy

Either way its the version of the gradle-intellij-plugin thats causing the problem here (I tested it with the new import method but without upgrading the version and everything works fine).